ESLintでエラーが発生する

## 起こっている問題 / The Problem
- ver 1.5.431のソースに対してyarn lintを実行すると
<pre>
nuxt.config.ts
  4:7  error  'webpack' is assigned a value but never used  @typescript-eslint/no-unused-vars
  4:7  error  'webpack' is assigned a value but never used  no-unused-vars

pages\parent.vue
  29:16  error  Replace `⏎··············'子供たちの学びを支援する様々なウェブサ イトを紹介しています。'⏎············` with `'子供たちの学びを支援する様々なウェ ブサイトを紹介しています。'`  prettier/prettier
</pre>
のエラーが発生する。

## スクリーンショット / Screenshot
<!-- バグであればdeveloper toolからコンソールも合わせて添付 -->
<!-- If it's a bug, attach a screenshot of the developer tool console -->

## 期待する見せ方・挙動 / Expected Behavior
- nuxt.config.ts についてはこの行を削除するとビルドが通らないので、暫定的に当該行に`// eslint-disable-line`をつけます。
- pages\parent.vue についてはyarn lint:fix実行後のソースに改めます。


## 起こっている問題の再現手段 / Steps to Reproduce
1. ver 1.5.431のソースに対してyarn lintを実行する

## 動作環境・ブラウザ / Environment
- Windows 10
- Node.js 12.18.1

--- a/nuxt.config.ts
+++ b/nuxt.config.ts
@@ -1,7 +1,6 @@
 import { Configuration } from '@nuxt/types'
 import { Configuration as WebpackConfiguration } from 'webpack'
 import i18n from './nuxt-i18n.config'
-const webpack = require('webpack')  // eslint-disable-line
 const purgecss = require('@fullhuman/postcss-purgecss')
 const autoprefixer = require('autoprefixer')
 const environment = process.env.NODE_ENV || 'development'

